loadtest/tests.go

```diff
@@ -151,6 +151,72 @@ func actionPost(c *EntityConfig) {
 	}
 }
 
+func actionJoinLeaveChannel(c *EntityConfig) {
+	// figure out my user id
+	var userId string
+	if user, resp := c.Client.GetUserByEmail(c.UserData.Email, ""); resp.Error != nil {
+		cmdlog.Errorf("Unable to get user by email. User: %v, Error: %v", c.UserData.Username, resp.Error.Error())
+	} else {
+		userId = user.Id
+	}
+
+	// pick a random team to join a channel from
+	team, _ := c.UserData.PickTeamChannel()
+	if team == nil {
+		cmdlog.Errorf("Unable to pick random team. User: %v", c.UserData.Username)
+		return
+	}
+	teamId := c.TeamMap[team.Name]
+
+	// get all the public channels on that team
+	var channelIdsInTeam []string
+	if channels, resp := c.Client.GetPublicChannelsForTeam(teamId, 0, 1000, ""); resp.Error != nil {
+		cmdlog.Errorf("Unable to get public channels for team. Team id: %v, User: %v, Error: %v", teamId, c.UserData.Username, resp.Error.Error())
+		return
+	} else {
+		channelIdsInTeam = make([]string, 0)
+		for _, channel := range channels {
+			channelIdsInTeam = append(channelIdsInTeam, channel.Id)
+		}
+	}
+
+	// get channels the user is already a member of on that team
+	var channelId *string
```

Comment:
Simpler and more idiomatic to just use `string` when the empty string isn't a valid value.